ml/requirements.txt

```diff
@@ -0,0 +1,11 @@
+scikit-learn
+matplotlib
+seaborn
```

Comment:
if possible, let's not include seaborn here, and instead, maybe our notebook dockerfile